projects/storefrontlib/src/cms-components/cart/cart-page-layout-handler.ts

```diff
@@ -8,14 +12,42 @@ import { PageLayoutHandler } from '../../cms-structure/page/page-layout/page-lay
   providedIn: 'root',
 })
 export class CartPageLayoutHandler implements PageLayoutHandler {
-  constructor(private cartService: CartService) {}
+  constructor(
+    private cartService: CartService,
+    private selectiveCartService: SelectiveCartService,
+    private featureConfig: FeatureConfigService
+  ) {}
```

Comment:
That is a breaking change. Make those services optional and follow the same steps like other deprecations in Spartacus.

Reply:
updated.